itest/test_manager.go

```diff
@@ -1,483 +0,0 @@
-package e2etest
-
-import (
-	"context"
-	"math/rand"
-	"os"
-	"sync"
-	"testing"
-	"time"
-
-	"github.com/babylonlabs-io/babylon/testutil/datagen"
-	bbntypes "github.com/babylonlabs-io/babylon/types"
-	btcctypes "github.com/babylonlabs-io/babylon/x/btccheckpoint/types"
-	btclctypes "github.com/babylonlabs-io/babylon/x/btclightclient/types"
-	bstypes "github.com/babylonlabs-io/babylon/x/btcstaking/types"
-	covcc "github.com/babylonlabs-io/covenant-emulator/clientcontroller"
-	covcfg "github.com/babylonlabs-io/covenant-emulator/config"
-	"github.com/babylonlabs-io/covenant-emulator/covenant"
-	signerCfg "github.com/babylonlabs-io/covenant-emulator/covenant-signer/config"
-	"github.com/babylonlabs-io/covenant-emulator/covenant-signer/keystore/cosmos"
-	signerMetrics "github.com/babylonlabs-io/covenant-emulator/covenant-signer/observability/metrics"
-	signerApp "github.com/babylonlabs-io/covenant-emulator/covenant-signer/signerapp"
-	"github.com/babylonlabs-io/covenant-emulator/covenant-signer/signerservice"
-	signerService "github.com/babylonlabs-io/covenant-emulator/covenant-signer/signerservice"
-	covdkeyring "github.com/babylonlabs-io/covenant-emulator/keyring"
-	"github.com/babylonlabs-io/covenant-emulator/remotesigner"
-	"github.com/babylonlabs-io/covenant-emulator/testutil"
-	"github.com/babylonlabs-io/covenant-emulator/types"
-	"github.com/btcsuite/btcd/btcec/v2"
-	"github.com/btcsuite/btcd/chaincfg"
-	"github.com/btcsuite/btcd/wire"
-	"github.com/cosmos/cosmos-sdk/crypto/keyring"
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
-	"github.com/stretchr/testify/require"
-	"go.uber.org/zap"
-)
-
-var (
-	eventuallyWaitTimeOut = 1 * time.Minute
-	eventuallyPollTime    = 500 * time.Millisecond
-	btcNetworkParams      = &chaincfg.SimNetParams
-
-	covenantKeyName = "covenant-key"
-	chainID         = "chain-test"
-	passphrase      = "testpass"
-	hdPath          = ""
-)
-
-type TestManager struct {
-	Wg               sync.WaitGroup
-	BabylonHandler   *BabylonNodeHandler
-	CovenantEmulator *covenant.CovenantEmulator
-	CovenanConfig    *covcfg.Config
-	CovBBNClient     *covcc.BabylonController
-	StakingParams    *types.StakingParams
-	baseDir          string
-}
-
-type TestDelegationData struct {
-	DelegatorPrivKey *btcec.PrivateKey
-	DelegatorKey     *btcec.PublicKey
-	SlashingTx       *bstypes.BTCSlashingTx
-	StakingTx        *wire.MsgTx
-	StakingTxInfo    *btcctypes.TransactionInfo
-	DelegatorSig     *bbntypes.BIP340Signature
-	FpPks            []*btcec.PublicKey
-
-	SlashingPkScript []byte
-	StakingTime      uint16
-	StakingAmount    int64
-}
-
-type testFinalityProviderData struct {
-	BabylonAddress sdk.AccAddress
-	BtcPrivKey     *btcec.PrivateKey
-	BtcKey         *btcec.PublicKey
-	PoP            *bstypes.ProofOfPossessionBTC
-}
-
-func StartManager(t *testing.T, useRemoteSigner bool) *TestManager {
```

Comment:
Hmm I am pretty sure we cannot delete this due to covenant emulator e2e test

Reply:
you are correct, I had searched for references in the code before removing it :thinking: 
Adding it back...

Reply:
found my problem with searching for references `//go:build e2e`